app/uk/gov/nationalarchives/omega/editorial/services/jms/StubData.scala

```diff
@@ -22,8 +22,15 @@
 package uk.gov.nationalarchives.omega.editorial.services.jms
 
 import uk.gov.nationalarchives.omega.editorial.models.MaterialReference.{ DescriptionOnly, LinkAndDescription, LinkOnly }
-import uk.gov.nationalarchives.omega.editorial.models.{ EditSet, EditSetEntry, EditSetRecord, LegalStatus, PlaceOfDeposit }
+import uk.gov.nationalarchives.omega.editorial.models._
+import com.google.inject.ImplementedBy
 
+import javax.inject.{ Inject, Singleton }
+
+@Singleton
+class StubDataImpl @Inject() extends StubData
+
+@ImplementedBy(classOf[StubDataImpl])
 trait StubData {
```

Comment:
Now sometimes we use the trait (e.g. `BaseControllerSpec`) and sometimes as an injected object (in `BaseISpec`).

I wonder if we shouldn't just always go with the latter.

Reply:
Some classes are using guice for dependency injection, and some are using trait inheritance hierarchies. IMHO it would be a major refactor to use guice everywhere. Most tests are also using traits.

Reply:
In that case, best to leave it.

And it may be more flexible this way.